Makefile

```diff
@@ -0,0 +1,232 @@
+BASE_PACKAGES := utils db settings visualization
+
+MODELS := biocro clm45 dalec ed gday jules linkages \
+	      lpjguess maat maespa preles sipnet
+
+
+MODULES := allometry assim.batch assim.sequential benchmark \
+	       data.atmosphere data.hydrology data.land \
+	       data.mining data.remote emulator meta.analysis \
+	       photosynthesis priors rtm uncertainty
+
+.PHONY: all ${BASE_PACKAGES} ${MODELS} ${MODULES}
+
+all: .install PEcAn.all ${BASE_PACKAGES} ${MODELS} ${MODULES}
```

Comment:
@robkooper I think that's @ashiklom's ultimate plan, but I told him to start with putting the packages in make and call it from build.sh, and then to transition more and more into make until build.sh is deprecated. I wouldn't hold up this PR on this specific point.